blog/_posts/2025-05-13-health-probe-improvements.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Smarter Load‑Balancer Health Probes for AKS"
+description: "We’re redesigning the default Azure Load Balancer probe for externalTrafficPolicy: Cluster services so your apps stay healthy, troubleshooting gets simpler, and you can finally turn off those extra NodePorts."
```

Comment:
this is still preview and default behavior is not changed yet. Should we add the link to AKS doc or AKS command to say how to enable this config?